zio-http/shared/src/main/scala/zio/http/codec/PathCodec.scala

```diff
@@ -458,6 +530,7 @@ object PathCodec          {
   private[http] sealed trait Opt
   private[http] object Opt {
     final case class Match(value: String)                     extends Opt
+    final case class MatchAny(values: Set[String])            extends Opt
```

Comment:
Good to use a set here. We can do even better than that by producing a finite state machine that produces true for a match, and false otherwise (due to the char-by-char nature of testing for string equality). But, separate issue and PR. 😄

Reply:
You mean some structure, where we have like `n` states, where `n` is max length of the alternatives and there is a way to progress to the next state with the right char, true if the string ends or false for any other char, right?